chapters/003-traceroute-aside/README.md

```diff
@@ -38,57 +38,57 @@ tcpdump: listening on eth0, link-type EN10MB (Ethernet), snapshot length 262144
     10.1.2.2.60765 > 10.1.1.2.33434: UDP, length 9
 18:38:04.884853 IP (tos 0xc0, ttl 64, id 55358, offset 0, flags [none], proto ICMP (1), length 65)
     10.1.2.3 > 10.1.2.2: ICMP time exceeded in-transit, length 45
-	IP (tos 0x0, ttl 1, id 32336, offset 0, flags [DF], proto UDP (17), length 37)
+    IP (tos 0x0, ttl 1, id 32336, offset 0, flags [DF], proto UDP (17), length 37)
```

Comment:
When these were indented differently than the lines surrounding them, my linter tried to automatically pull them back to the margin. I figured indenting them some at leas made it clear where a single packet was being examined, so I aligned them with the their surrounding paragraphs.